feat(new): Add documentation for the new Update and Replace Metadata API Definitions (#371)
vectara/vectara-docs: www/docs/api-reference/admin-apis/create-corpus.md

## Filter Attribute

> In order to reference metadata in [filter expressions](/docs/learn/metadata-search-filtering/filter-overview), you
> declare the referenceable attributes at creation time in the **filter
> attributes**. If you already created a corpus, use the 
> [Replace Filter Attributes API](/docs/api-reference/admin-apis/corpus/replace-filter-attributes). 

Reviewer: ```suggestion
In order to reference metadata in [filter expressions](/docs/learn/metadata-search-filtering/filter-overview),  
you must define the **filter attributes** at the time of corpus creation. If  
you already created a corpus, use the [Replace Filter Attributes API](/docs/api-reference/admin-apis/corpus/replace-filter-attributes).  
```

Non-blocking
